qiskit_ibm_runtime/fake_provider/__init__.py

```diff
@@ -135,6 +135,7 @@
     FakeKyoto
     FakeLagosV2
     FakeLimaV2
+    FakeAltLimaV2
```

Comment:
Looks like `FakeAltLimaV2` is only created to include fractional gates + control flow. So why not use a more meaningful name, like `FakeFractionalBackend` ?